### ISSUE TITLE
No indicator for Spend Tx progress

### ISSUE THREAD

**Issue by JSwambo** [revault/revault-gui#86]
As a manager, after I broadcast a Spend Tx, the Tx shows up under 'pending spend transactions' but there's no info to tell me when it is expected to be confirmed. Perhaps we should put `pending unvault: 0/CSV`, `pending spend: 0/6 confirmations` or something?

**Comment by darosior** [comment]
Agreed, though we should use a much simpler terminology than `CSV` and "Unvault" in my opnion (see #88). Maybe:
1. "Pending spending attempt: currently locked. ETA: 2h and a half" (Accounting for a CSV of 12 and a 3-blocks confirmation)
2. "Pending spending attempt: now unlocked. ETA: half a hour" (3 blocks confirmation"
